File: src/main/java/net/hectus/neobb/turn/default_game/flower/TDirt.java
package net.hectus.neobb.turn.default_game.flower;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.turn.DummyTurn;
import net.hectus.neobb.turn.default_game.block.BlockTurn;
import org.bukkit.block.Block;

public class TDirt extends BlockTurn implements DummyTurn {
    public TDirt(NeoPlayer player) { super(player); }
    public TDirt(Block data, NeoPlayer player) { super(data, player); }

    @Override
    public int cost() {
        return 2;
    }
}

File: src/main/kotlin/net/hectus/neobb/modes/turn/default_game/flower/TDirt.kt
package net.hectus.neobb.modes.turn.default_game.flower

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.modes.turn.ComboTurn
import net.hectus.neobb.modes.turn.default_game.block.BlockTurn
import net.hectus.neobb.player.NeoPlayer
import org.bukkit.block.Block

class TDirt(data: Block?, cord: Cord?, player: NeoPlayer?) : BlockTurn(data, cord, player), ComboTurn {
    override val cost: Int = 2
}
